### DISCUSSION TITLE
Reselecting Exercise Type

### DISCUSSION THREAD

**Discussion by gabriellecozart** [SethClydesdale/genki-study-resources#185] [Ideas]
Occasionally I get into an issue where I can't select my exercise type. There's no prompt for the lesson and I can't find a way to change it. It would be cool if there was a button on the bottom, similar to the "hide furigana" button that allows you to select and change it.

If this is a limitation of the current lesson (unsure) it's hard to know as the user that it's not supported. A grayed out "change exercise type" button would be helpful. You could even have a roll over that explains why it's not available. For instance, I am on [第4課 - Useful Expressions: Time Words](https://sethclydesdale.github.io/genki-study-resources/lessons/lesson-4/grammar-4/) exercise where it's drag and drop, but I'd rather see English and type the answer in Japanese. 

Side note, this resource is super amazing and I just learned what anki is and omg I wish I had all of  this when I was studying in school <3

**Comment by SethClydesdale** [comment]
Hello @gabriellecozart! This is indeed a limitation of the 2nd edition resources. During the first development cycle, exercises only had a single type; drag & drop for vocab, multi-choice for practice, written for workbook, etc. It wasn't until I worked on the 3rd edition resources that the option for multiple exercise types and full furigana support was added.

From what I recall, I believe this functionality is only present in the dictionary and custom exercises for the 2nd edition since they both use the same script. Fortunately the 3rd edition is nigh identical to the 2nd, so if you like, you can switch to the 3rd edition at any time for more exercise types: https://sethclydesdale.github.io/genki-study-resources/lessons-3rd/lesson-4/vocab-10/?start=3

Switching `/lessons/` to `/lessons-3rd/` (and vice versa) in the URL will *usually* lead to the same exercise on that edition. I say usually, because, there were some changes in order/structure here and there, so there are times you may have to reposition via the exercise list or use the [quick search](https://sethclydesdale.github.io/genki-study-resources/lessons-3rd/?search=time%20words#quick-search-exercises).

Hope this helps!

**Reply by gabriellecozart** [reply]
Thank you for the info!
